src/test/java/seedu/address/logic/parser/TagInfoCommandParserTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.TagInfoCommand;
+import seedu.address.logic.commands.exceptions.CommandException;
+
+public class TagInfoCommandParserTest {
+
+    private TagInfoCommandParser parser = new TagInfoCommandParser();
+
+    @Test
+    public void assertAllValuesCorrect() throws CommandException {
```

Comment:
I think the name of the test method should be execute_EmptyAddressBook_Success(use underscore), this applies to all your method name below.